_layouts/project.html.haml

```diff
@@ -103,6 +103,8 @@
                       %a(href="https://source.jboss.org/browse/narayana") FishEye
                 %li
                   %a(href="https://issues.jboss.org/browse/JBTM") Issue Tracker
+                %li
+                  %a(href="/lra/certifications") Certifications
```

Comment:
the link needs to be relative (ie no leading slash) and also include index.html, ie:

`%a(href="lra/certifications/index.html") Certifications`

Similarly for all the other references to the Certifications menu item.